vanilla-framework/vanilla-framework-react
# Update Storybook and components to Vanilla 1.8.0

## Done

- Updated Storybook app to use Vanilla 1.8.0 and fixed associated visual bugs
- Fixed a bug where empty MatrixItems were displaying a small empty box instead of nothing
- Rewrote MediaObject to be a little more flexible and use new markup
- Fixed a small bug with NavigationBanner if using a text title instead of an image logo
- Added close button to Notification component
- Converted Switch component to the new checkbox version
- Fixed the MediaObject in the expandable Table component based on the changes made for 1.8.0
- Updated tests with Vanilla 1.8.0 changes (see #135)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8102/](http://0.0.0.0:8102/)
- Go through all the components and check that they look as they should in the [Vanilla examples](https://vanilla-framework.github.io/vanilla-framework/)
- Check that Matrix, Navigation, Notification, Switch and MediaObject function properly

## Issue / Card

Fixes #129
